R/package-auth.R

```diff
@@ -18,7 +18,8 @@
 #' @export
 db_host <- function(id = NULL, prefix = NULL) {
   if (is.null(id) && is.null(prefix)) {
-    host <- Sys.getenv("DATABRICKS_HOST")
+    #host <- Sys.getenv("DATABRICKS_HOST")
+    host=db_get_param('dbHost')
```

Comment:
ideally the fetching of param would only be offloaded in circumstance of using `.databrickscfg`, at least that's my current thinking.